src/Compilers/CSharp/Portable/Syntax/InternalSyntax/SyntaxToken.MissingTokenWithTrivia.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax
 {
     internal partial class SyntaxToken
     {
-        internal class MissingTokenWithTrivia : SyntaxTokenWithTrivia
+        internal sealed class MissingTokenWithTrivia : SyntaxTokenWithTrivia
```

Comment:
did this to amke it easier to prove certain invariants about missing tokens to myself.